src/assets/css/_sass/elements/_navs.scss

```diff
@@ -3,3 +3,40 @@
   Navs
 
 */
+
+#nav-header {
+  position: absolute;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  padding: 30px 50px;
+  width: 100%;
+}
+
+#main-nav {
+  display: flex;
+  flex-direction: row; // width: 400px;
+  justify-content: flex-start;
+  -webkit-box-shadow: 3px 4px 5px 0px rgba(0, 0, 0, 0.75);
+  -moz-box-shadow: 3px 4px 5px 0px rgba(0, 0, 0, 0.75);
+  box-shadow: 3px 4px 5px 0px rgba(0, 0, 0, 0.75);
```

Comment:
box-shadow should be replaced by the box-shadow mixin, if you find that the designs are all over the place for the shadows, ask yourself if the designs really need to be like that? what happens when you make them all the same, etc. Most of the time no one is going to care.